hl7_2_fhir/huisartswaarneemgegevens/6.10.1.3/professionele_samenvatting/payload/QUPC_IN990002NL-2-BundleGeneralPractitioner.xsl

```diff
@@ -1395,7 +1399,10 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                             <valueBoolean value="{hl7:value/@value}"/>
                         </xsl:when>
                         <xsl:when test="hl7:value">
-                            <xsl:message>Niet-ondersteund datatype <xsl:value-of select="@xsi:type"/></xsl:message>
+                            <xsl:call-template name="util:logMessage">
+                                <xsl:with-param name="level" select="$logWARN"/>
+                                <xsl:with-param name="msg">Niet-ondersteund datatype <xsl:value-of select="@xsi:type"/></xsl:with-param>
```

Comment:
Translate `msg` to English, as above `msg` is also in English (?)

Reply:
Agree, formally out of scope for this ticket, but updated it